src/comp/comp.py

```diff
@@ -24,48 +29,48 @@ def __repr__(self):
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with 'D':
 print("Starts with D:")
-a = []
+a = [i.name for i in humans if i.name[0] == "D"]
 print(a)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name ends in "e".
 print("Ends with e:")
-b = []
+b = [i.name for i in humans if i.name[-1] == "e"]
```

Comment:
You can also use `endsWith()` in comp.py to determine if string ends with particular letter.